src/cljs/coffee_re_frame/effects.cljs

```diff
@@ -37,3 +37,33 @@
   (.vibrate js/navigator duration))
 
 (re-frame/reg-fx :vibrate vibrate-effect)
+
+(def current-lock (atom nil))
+
+(defn lock-brightness []
+  (when-let [wakeLock (.-wakeLock js/navigator)]
+    (->
+     (.request wakeLock "screen")
+     (.then (fn [new-lock]
+              (js/console.log "Locking screen brightness: " new-lock)
```

Comment:
[nit]: Do we not want these to be conditional debug logs? Makes little sense to log them in prod